Source/WebKit/UIProcess/Launcher/cocoa/ExtensionProcess.mm

```diff
@@ -69,8 +75,41 @@
 {
     NSError *error = nil;
     RetainPtr<BEProcessCapabilityGrant> grant;
-    WTF::switchOn(m_process, [&] (auto& process) {
+    WTF::switchOn(m_process, [&] (RetainPtr<BEWebContentProcess> process) {
```

Comment:
You don't need so many lambdas as far as I know. As I told you last time, you can rely on `auto`:
```
WTF::switchOn(m_process, [&] (auto& process) {
    grant = [process grantCapability:capability error:&error];
}, [](const RetainPtr<_SEExtensionProcess>&) { });
```